buildconfig/config_emsdk.py

```diff
@@ -7,85 +7,115 @@
 
 from distutils.sysconfig import get_python_inc
 
-configcommand = os.environ.get('SDL_CONFIG', 'sdl-config',)
-configcommand = configcommand + ' --version --cflags --libs'
-localbase = os.environ.get('LOCALBASE', '')
-if os.environ.get('PYGAME_EXTRA_BASE'):
-    extrabases = os.environ['PYGAME_EXTRA_BASE'].split(':')
+configcommand = os.environ.get(
+    "SDL_CONFIG",
+    "sdl-config",
```

Comment:
Shouldn't this be `sdl2-config`? That is the case on linux atleast and I'm not sure whether it's the the same on emscripten

Reply:
indeed though it probably does not have any effect. that config file could use a big cleanup i copied too much things from unix cfg.